health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -777,13 +919,391 @@ export const UICustomizations = {
+      }
+
+      data.params.names = name;
+      data.params.sortOrder = "DESC";
+      data.params.sortBy = "lastModifiedTime";
+
+      data.params.phone = phone;
+
+      data.params.roles = rolesString;
+      data.params.tenantId = Digit.ULBService.getCurrentTenantId();
+      cleanObject(data.params);
+      delete data.params.roleschosen;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)